### DISCUSSION TITLE
Multiple fill modules

### DISCUSSION THREAD

**Discussion by Drllap** [starship/starship#5691] [Q&A]
I want to split the module line into sections, similar to what is shown in the [Fill module example](https://starship.rs/config/#fill), except that I want to have a different style (background color) for each section. Can I define multiple fill modules or is there another way of doing this?

**Comment by davidkna** [comment]
It's not possible to create different configurations for the same module, but you might be able to style them from the top-level format string e.g. `[$fill](bg:blue)...[$fill](bg:red)`.

**Reply by Drllap** [reply]
I tried that ,doesn't appear to work

**Reply by sharpjs** [reply]
It works if you configure `fill` like:

```toml
[fill]
style = 'none'
```
